pkg/bundle/internal/truststore/types_test.go

```diff
@@ -146,3 +143,60 @@ func Test_certAlias(t *testing.T) {
 		t.Fatalf("expected alias to be %q but got %q", expectedAlias, alias)
 	}
 }
+
+func Test_manual_read_JKS(t *testing.T) {
+	t.Skip("these tests require installation of Java to run")
+
+	tests := map[string]struct {
+		keytool string
+	}{
+		"Java 21": {
+			keytool: "/usr/lib/jvm/java-21-openjdk-amd64/bin/keytool",
+		},
+		"Java 17": {
+			keytool: "/usr/lib/jvm/java-17-openjdk-amd64/bin/keytool",
+		},
+		"Java 11": {
+			keytool: "/usr/lib/jvm/java-11-openjdk-amd64/bin/keytool",
+		},
+		"Java 8": {
+			keytool: "/usr/lib/jvm/java-8-openjdk-amd64/bin/keytool",
+		},
+	}
```

Comment:
suggestion: I don't know if the differing versions of keytool would meaningfully change this - it's not the same as running different versions of Java AFAIK. Plus, these paths assume amd64 (and so wouldn't work for me)

I'd vote to remove the different versions here and, like in my other suggestion, just run this test if keytool can be found on $PATH.

Reply:
> it's not the same as running different versions of Java AFAIK.

Important note! Hmmm, then I must find a better way to test this. 🤔

Reply:
@SgtCoDFish, I hacked together a test that actually runs Java against a Go HTTP server using a self-signed certificate. And it seems to work on my machine for all the LTS Java versions: 8 (EOL), 11, 17 and 21. Do you have Java installed? It would be nice if you could also test! 😅